### PR TITLE
Todo app using React-Redux

### PR DESCRIPTION
**Pull request by malinvan**
Link to my deployed project:
https://malins-redux-todo-app.netlify.app

My reflections on how this week's project turned out:
The functionality went fine, I'm still getting used to the styled components though.
I'm not super happy about the styling.

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,68 @@
-import React from 'react'
```
Comment (sssylvasss) [inline code/src/App.js:1]: Maybe separated the different kinds of imports?

File: code/src/components/AddTodo.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { todos } from '../reducers/todos';
+import styled from 'styled-components';
+// import { AddBtn } from './AddBtn'
```
Comment (sssylvasss) [inline code/src/components/AddTodo.jsx:5]: Delete code that is not used

File: code/src/components/TodoList.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  color: #D81159;
+  border: none;
+`;
+
+const TimeStamp = styled.p`
+  color: gray;
+`;
+
+const Input = styled.input`
+
```
Comment (sssylvasss) [inline code/src/components/TodoList.jsx:41]: delete extra space/line:)

File: code/src/reducers/todos.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+export const todos = createSlice({
+  name: 'todos',
+  initialState: {
+    items: [
+      { id: 1, description: 'Buy groceries', isComplete: true },
+      { id: 2, description: 'Take a walk', isComplete: false },
+      { id: 3, description: 'Make dinner', isComplete: false },
+      { id: 4, description: 'Clean', isComplete: false },
+      { id: 5, description: 'Code', isComplete: false },
+      { id: 6, description: 'Go for a run', isComplete: false }
```
Comment (sssylvasss) [inline code/src/reducers/todos.js:13]: Maybe empty the array? or is it more for show?

**Review by sssylvasss — COMMENTED** [review]
Hej! 
I have not much to add. Nice and clean code as always! 
Everything works when I check it. It's  nice that the completed tasks moves down.

:)
Ylva

File: code/src/App.js (hunk 1 above)
Comment (KarolCislowski) [reply code/src/App.js:1]: Great Idea 🤓

File: code/src/components/AddTodo.jsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const InputContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
```
Comment (KarolCislowski) [inline code/src/components/AddTodo.jsx:18]: Both are the same 👽 👽

File: code/src/components/CompletedItems.jsx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+const DeleteBtn = styled.button`
+  background-color: black;
+  color: #D81159;
+  border: none;
+`;
+
+export const CompletedItems = () => {
+  const items = useSelector((store) => store.todos.items.filter(item => item.isComplete));
+  const dispatch = useDispatch();
+  return (
```
Comment (KarolCislowski) [inline code/src/components/CompletedItems.jsx:56]: separate different operation with empty lines 😸

**Review by KarolCislowski — APPROVED** [review]
Great app with a very nice interface, great job!

**Pull request closed without merging**